analysis/cnvs/query_match_nch_clinvar_cnvs.ipynb

```diff
@@ -387,11 +391,60 @@
     }
    ],
    "source": [
-    "dv = pd.read_csv(\"cnv_data/ClinVar-CNVs-normalized.csv\")\n",
+    "dv = pd.read_csv(\"cnv_data/ClinVar-CNVs-normalized.csv.gzip\", compression=\"gzip\")\n",
     "print(\"Rows:\", len(dv))\n",
     "dv.head()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Helper function to populate chr field for normalized variants where this field is missing\n",
+    "import re\n",
+    "import requests\n",
+    "\n",
+    "def get_chr_from_cytogenetic(cyto: str) -> str:\n",
+    "    \"\"\"\"Get chromosome number/letter from cytogenetic string\n",
+    "    \n",
+    "    :param cyto: The cytogenetic string\n",
+    "    :return: The chromosome number or letter\"\"\"\n",
+    "    return re.split(r'(?=[pq])', cyto)[0]\n",
+    "\n",
+    "def get_chr_from_name(name: str) -> str:\n",
+    "    \"\"\"Get chromosome number/string from variant name\n",
+    "    \n",
+    "    :param name: The variant name\n",
+    "    :return: The chromosome number or letter\"\"\"\n",
+    "    name = name.split(\":\")[0]\n",
+    "    query_string = \"https://services.genomicmedlab.org/seqrepo/1/metadata/refseq%3A\" + name\n",
```

Comment:
Don't use REST API, please use SeqRepo Python API. Pointed you to REST for quick test